handlers/vm/lxc/lxc.go

```diff
@@ -85,13 +96,21 @@ func (h *LxcHandler) MergeArgs(dst model.ResourceTemplate, args []string) error
 
 	flags := flag.NewFlagSet("lxc template", flag.ContinueOnError)
 	var vcpu, mem int
+	var authType, sshPubkey string
 	flags.IntVar(&vcpu, "vcpu", int(mdst.MinVcpu), "")
 	flags.IntVar(&mem, "memory_gb", int(mdst.MinMemoryGb), "")
+	defAuth := model.AuthenticationType_name[int32(mdst.AuthenticationType)]
+	flags.StringVar(&authType, "authentication_type", defAuth, "")
+	flags.StringVar(&sshPubkey, "ssh_public_key", mdst.SshPublicKey, "")
 	if err := flags.Parse(args); err != nil {
 		return err
 	}
 	mdst.Vcpu = int32(vcpu)
 	mdst.MemoryGb = int32(mem)
+	format := model.AuthenticationType_value[strings.ToUpper(authType)]
```

Comment:
Missing the validation  if ``--authtication_type`` gets unknown value.

Reply:
Added validation of authtication_type to lxc.go and qemu.go.